client/app/components/composites/ListingCard/ListingCard.css

```diff
@@ -105,6 +105,13 @@
   padding-top: 5px;
 }
 
+.orderTypeLabel {
+  flex-basis: 68%;
+  font-size: var(--ListingCard_fontSizePrice);
+  font-weight: var(--ListingCard_fontWeightMedium);
+  line-height: 1em;
+}
+
```

Comment:
You have a sharp eye :)
I wanted to keep them separated since those styles are not yet final - design review needed.
If we end up merging, I'll suggest a renaming (priceWrapper -> orderTypeWrapper) since price is just one special case of order type.